liquidator/OneInchExchange.js

```diff
@@ -1,10 +1,8 @@
 const Token = artifacts.require("ExpandedERC20");
 const OneSplit = artifacts.require("OneSplit");
+const { toBN, toWei } = web3.utils;
 
-// As defined in 1inch
-const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
-
-const ONE_SPLIT_ADDRESS = "0xC586BeF4a0992C495Cf22e1aeEE4E446CECDee0E";
+const { ONE_SPLIT_ADDRESS, ETH_ADDRESS } = require("./constants");
 
 class OneInchExchange {
```

Comment:
I know it was before this PR but the constructor params in this are inconsistent with the rest of the repo. Like I mentioned in [this](https://github.com/UMAprotocol/protocol/pull/1855/files#r471949765) comment, nowhere else do we feed in objects like this.

Also The convention is always that the logger comes first. See [here](https://github.com/UMAprotocol/protocol/blob/dccb2feb9e5aa883648ee7dab4896c1bc6d92694/liquidator/liquidator.js#L22) or [here](https://github.com/UMAprotocol/protocol/blob/dccb2feb9e5aa883648ee7dab4896c1bc6d92694/liquidator/liquidator.js#L22).

Reply:
I understand that you would like consistencies, but the example you're linking has been merged into the repo, and converting that style back to conform to the rest of the repo will result in more diffs

Reply:
ye, ok you can leave this one for now and then add the sweeping changes in #1869